app/src/main/java/org/oppia/android/app/administratorcontrols/AdministratorControlsViewModel.kt

```diff
@@ -64,15 +67,26 @@ class AdministratorControlsViewModel @Inject constructor(
   private fun processAdministratorControlsList(
     deviceSettings: DeviceSettings
   ): List<AdministratorControlsItemViewModel> {
-    val itemViewModelList: MutableList<AdministratorControlsItemViewModel> = mutableListOf(
-      AdministratorControlsGeneralViewModel()
-    )
-    itemViewModelList.add(
-      AdministratorControlsProfileViewModel(
-        routeToProfileListListener,
-        loadProfileListListener
+
+    val itemViewModelList: MutableList<AdministratorControlsItemViewModel>
+
+    if (enableEditAccountsOptionsUi.value) {
+      itemViewModelList = mutableListOf(
+        AdministratorControlsGeneralViewModel(),
+        AdministratorControlsProfileViewModel(
+          routeToProfileListListener,
+          loadProfileListListener
+        )
       )
-    )
+    }else{
```

Comment:
There should be spaces around the 'else'. See https://developer.android.com/kotlin/style-guide#whitespace.

Further, it seems that your lint tests are failing. I suggest setting these up as a pre-push hook to ensure that lint issues are addressed before code review (this helps make things run more smoothly). See: https://github.com/oppia/oppia-android/wiki/Contributing-to-Oppia-android#install-oppia-android (you need to run the setup script in your local repository).